### PR TITLE
Simplified true/false

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

Removed unnecessary true/false

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/core/Mage/Adminhtml/Block/Catalog/Category/Tree.php [hunk 1]
```diff
@@ -242,7 +242,7 @@ protected function _getNodeJson($node, $level = 0)
         $allowMove = $this->_isCategoryMoveable($node);
         $item['allowDrop'] = $allowMove;
         // disallow drag if it's first level and category is root of a store
-        $item['allowDrag'] = $allowMove && (($node->getLevel()==1 && $rootForStores) ? false : true);
+        $item['allowDrag'] = $allowMove && (!($node->getLevel() == 1 && $rootForStores));
```
Comment (kiatng) [inline app/code/core/Mage/Adminhtml/Block/Catalog/Category/Tree.php:245]: Unnecessary brackets?

```suggestion
$item['allowDrag'] = $allowMove && !($node->getLevel() == 1 && $rootForStores);
```

File: app/code/core/Mage/Core/Model/Config.php [hunk 2]
```diff
@@ -833,7 +833,7 @@ protected function _loadDeclaredModules($mergeConfig = null)
             $moduleDepends[$moduleName] = array(
                 'module'    => $moduleName,
                 'depends'   => $depends,
-                'active'    => ('true' === (string)$moduleNode->active ? true : false),
+                'active'    => ('true' === (string)$moduleNode->active),
```
Comment (kiatng) [inline app/code/core/Mage/Core/Model/Config.php:836]: Me think is this:

```suggestion
                'active'    => (bool)(string)$moduleNode->active,
```

File: app/code/core/Mage/Admin/Model/User.php [hunk 3]
```diff
@@ -272,7 +272,7 @@ public function deleteFromRole()
     public function roleUserExists()
     {
         $result = $this->_getResource()->roleUserExists($this);
-        return (is_array($result) && count($result) > 0) ? true : false;
+        return (is_array($result) && count($result) > 0);
```
Comment (kiatng) [inline app/code/core/Mage/Admin/Model/User.php:275]: Is it better without brackets?

```suggestion
        return is_array($result) && count($result) > 0;
```

File: app/code/core/Mage/Core/Model/Config.php (hunk 2 above)
Comment (kiatng) [inline app/code/core/Mage/Core/Model/Config.php:836]: My preference:

```suggestion
                'active'    => (string)$moduleNode->active === 'true',
```

File: app/code/core/Mage/Admin/Model/User.php [hunk 4]
```diff
@@ -294,7 +294,7 @@ public function add()
     public function userExists()
     {
         $result = $this->_getResource()->userExists($this);
-        return (is_array($result) && count($result) > 0) ? true : false;
+        return (is_array($result) && count($result) > 0);
```
Comment (kiatng) [inline app/code/core/Mage/Admin/Model/User.php:297]:
```suggestion
        return is_array($result) && count($result) > 0;
```

File: app/code/core/Mage/Api/Model/User.php [hunk 5]
```diff
@@ -176,7 +176,7 @@ public function deleteFromRole()
     public function roleUserExists()
     {
         $result = $this->_getResource()->roleUserExists($this);
-        return ( is_array($result) && count($result) > 0 ) ? true : false;
+        return (is_array($result) && count($result) > 0);
```
Comment (kiatng) [inline app/code/core/Mage/Api/Model/User.php:179]:
```suggestion
        return is_array($result) && count($result) > 0;
```

File: app/code/core/Mage/Api/Model/User.php [hunk 6]
```diff
@@ -198,7 +198,7 @@ public function add()
     public function userExists()
     {
         $result = $this->_getResource()->userExists($this);
-        return ( is_array($result) && count($result) > 0 ) ? true : false;
+        return (is_array($result) && count($result) > 0);
```
Comment (kiatng) [inline app/code/core/Mage/Api/Model/User.php:201]:
```suggestion
        return is_array($result) && count($result) > 0;
```

**Review by kiatng — CHANGES_REQUESTED** [review]

**Review by kiatng — APPROVED** [review]
Thank you.

**Review by kkrieger85 — APPROVED** [review]

**Review by colinmollenhour — APPROVED** [review]

**Pull request merged**